data/styles/clear/include/Icons.mapcss

```diff
@@ -1237,6 +1237,8 @@ area|z17-[leisure=swimming_pool][access=private]
 {icon-image: zero-icon.svg;}
 node|z15-[leisure=golf_course],
 area|z15-[leisure=golf_course],
+node|z15-[leisure=miniature_golf],
```

Comment:
I think z15 is too early - its a leisure activity similar to bowling, cinema or night club which start from z16/z17

Reply:
fair, playgrounds and public pools which are around the same size render at z17. moved 👍️